src/pages/copylist/CopyListPage.tsx

```diff
@@ -4,7 +4,7 @@ import { Accordion, AccordionSet, Layout, Loading } from '@folio/stripes/compone
 import { TitleManager } from '@folio/stripes/core';
 import { useHistory, useParams } from 'react-router-dom';
 import { HTTPError } from 'ky';
-import { useCreateList, useInitRefresh, useListDetails, useMessages } from '../../hooks';
+import {useCreateList, useInitRefresh, useListDetails, useMessages, useRecordTypeLabel} from '../../hooks';
```

Comment:
Any reason the spacing was removed? If the line is too long with it you can split the import across lines